src/orion/core/__init__.py

```diff
@@ -117,7 +112,7 @@ def define_database_config(config):
     database_config.add_option(
         "host",
         option_type=str,
-        default=default_host,
+        default="orion_db.pkl",
```

Comment:
Default should be https://github.com/Epistimio/orion/blob/develop/src/orion/core/io/database/pickleddb.py#L25

Reply:
Maybe but trying to import that file in the config function leads to an import loop and fails.

Reply:
But I found out that I can just reach into DIRS locally and use that to build the default path so that works.

Reply:
Actually you can forget about this default. I prefer Olivier's idea to set it to None by default and leave it to the DB class to set it to DEFAULT if it is None. Could you please do this in this PR?

Reply:
Ok

Reply:
It actually doesn't work and sets the default db to the string "None" if I put `None` there.

Reply:
Yes that's what I meant by 'leave it to the DB class to set it to DEFAULT if it is None. Could you please do this in this PR?'

You would need to adjust the code here: https://github.com/Epistimio/orion/blob/develop/src/orion/core/io/database/pickleddb.py#L100

Reply:
The 'None' string was due to Configuration, but I've adjusted to make the default an empty string and the database classes will replace that with their built-in defaults.